docs/WebEditorBasics/WebPlayMode.md

```diff
@@ -0,0 +1,53 @@
+---
+id: WebPlayMode
+title: Web Play Mode
+---
+
+Don't always have access to a device? Waiting for a battery to recharge? No problem!
+
+You can continue to build with Enklu Cloud's blazing fast iteration times by using Web Play Mode to test your experiences.
+
+// TODO: Liam's awesome gif here.
+
+## What is Web Play Mode?
+
+Web Play Mode is where your experience comes to life, without the extra weight of the editor tools. 
+On device, this is the default mode that experiences load into. From Enklu Cloud, you can switch to play mode at any time while developing an experience.
+
+This can be useful for getting a first person view of the experience and testing any potential scripting that occurs at the start of a session.
```

Comment:
"first person" needs a hyphen ("first-person")

Reply:
instead of "and testing any potential...," it would be better to say "and for testing any potential..."